### PR TITLE
Add warning to youtube extractor

### PR DESCRIPTION
**Pull request by retrouser955**
## Changes
Adds a warning to YouTube extractor via `process.emitWarning()`, warning the user of some instability nature of Youtube Extractor and recommending discord-player-youtubei

## Status

- [x] These changes have been tested and formatted properly.
- [ ] This PR includes only documentation changes, no code change.
- [ ] This PR introduces some Breaking changes.

### CODE REVIEW

**Review by twlite — COMMENTED** [review]
we should probably document this as well, and is this extractor listed in showcase?

**Comment by retrouser955** [comment]
Yes definitely. One slight problem. I have no idea how to modify the documentation. I'll try to tweak some settings and hope it works.

File: README.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ $ npm install --save ffmpeg-binaries
 
 YouTube streaming is not supported without installing one of the following package. If you want to add support for YouTube playback, you need to install a streaming library. This step is not needed if you do not plan on using youtube source.
 
+*The default Youtube streaming appears to be unstable due to recent changes. It is recommend to install [`discord-player-youtubei`](https://npm.im/discord-player-youtubei)*
```
Comment (twlite) [inline README.md:90]:
```suggestion
* The default Youtube streaming appears to be unstable due to recent changes. It is recommend to install [`discord-player-youtubei`](https://npm.im/discord-player-youtubei)*
```

File: apps/website/src/data/showcase.ts [hunk 2]
```diff
@@ -194,6 +194,11 @@ export const ShowcaseResource: IShowcase = {
             name: 'discord-player-tidal',
             description: 'An unofficial discord-player extractor to add support for tidal source.',
             url: 'https://npm.im/discord-player-tidal'
+        },
+        {
+            name: "discord-player-youtube", 
+            description: "Super charging your bot. dpy is a tiny extractor module to test the validity of youtubei.js in discord-player version 7.",
```
Comment (twlite) [inline apps/website/src/data/showcase.ts:200]: Maybe make it more informative such that people understand this is better than default youtube extractor.

File: apps/website/src/pages/guide/_guides/welcome/welcome.mdx [hunk 3]
```diff
@@ -94,6 +94,8 @@ $ npm install --save ffmpeg-binaries
 
 YouTube streaming is not supported without installing one of the following package. If you want to add support for YouTube playback, you need to install a streaming library. This step is not needed if you do not plan on using youtube source.
 
+*The default Youtube streaming appears to be unstable due to recent changes. It is recommend to install [`discord-player-youtubei`](https://npm.im/discord-player-youtubei)*
```
Comment (twlite) [inline apps/website/src/pages/guide/_guides/welcome/welcome.mdx:97]:
```suggestion
* The default Youtube streaming appears to be unstable due to recent changes. It is recommend to install [`discord-player-youtubei`](https://npm.im/discord-player-youtubei)*
```

File: packages/extractor/src/extractors/YoutubeExtractor.ts [hunk 4]
```diff
@@ -46,6 +46,8 @@ export class YoutubeExtractor extends BaseExtractor<YoutubeExtractorInit> {
             this._ytLibName = name;
         }
 
+        process.emitWarning("YoutubeExtractor uses scraping-based streaming libraries which is known to be unstable at times.\nAn alternative is to use discord-player-youtubei https://github.com/retrouser955/discord-player-youtubei")
```
Comment (twlite) [inline packages/extractor/src/extractors/YoutubeExtractor.ts:49]:
```suggestion
        process.emitWarning("YoutubeExtractor uses scraping-based streaming libraries which is known to be unstable at times. An alternative is to use discord-player-youtubei https://github.com/retrouser955/discord-player-youtubei");
```

Could we also link the github issue here?

**Review by twlite — CHANGES_REQUESTED** [review]

**Review by twlite — APPROVED** [review]

**Pull request merged**